neo/rawio/phyrawio.py

```diff
@@ -132,6 +133,9 @@ def _parse_header(self):
         for index, clust_id in enumerate(clust_ids):
             spiketrain_an = seg_ann['spikes'][index]
 
+            # Add cluster_id annotation
+            spiketrain_an['cluster_id'] = clust_id
+
```

Comment:
I am OK with this but we already have have this information in `self.header['spike_channels']['id']`  no ?
Maybe the SpikeTrainProxy object is not transmiting this info correctly in  basefromrawio.

Reply:
Just to be clear, the id in `unit_name` and the `id` annotation is the same as `cluster_id`, but the pair is `str` and the 'cluster_id' is `int`. They indicate the id of the cluster and _are not necessarily sequential_ (e.g. `id` 56, 67, 87 are missing because those ids point to empty clusters).